js/okx.js

```diff
@@ -615,6 +615,17 @@ module.exports = class okx extends Exchange {
                     'method': 'privateGetAccountBills', // privateGetAccountBillsArchive, privateGetAssetBills
                 },
                 // 1 = SPOT, 3 = FUTURES, 5 = MARGIN, 6 = FUNDING, 9 = SWAP, 12 = OPTION, 18 = Unified account
+                'fetchOrder': {
+                    'method': 'privateGetTradeOrder', // privateGetTradeOrdersAlgoHistory
+                    'algoOrderTypes': {
```

Comment:
Now that we have this duplication of `algoOrderTypes` in multiple methods, let's move that one level higher in the options-hierarchy, in other words, let's list them out just once in the `options['algoOrderTypes']` and remove the duplication from `options['fetchOrder']`, `options['fetchOpenOrders']`, etc